### ISSUE TITLE
movie.nfo fanart XML is not correct 

### ISSUE THREAD

**Issue by nathan951** [tinyMediaManager/tinyMediaManager#115]
Couldn't figure-out why my TMM movies edited/saved with movie.nfo didn't display fan art, until i compared it with kodi created .nfo files.

To test check save "movie.nfo" 

fan art will will be saved as

`<fanart>http://image.tmdb.org/t/p/original/id2qZADRqKEGFxf7yeHLp4tHaEo.jpg</fanart>`

where it should be saved as:

   ```
<fanart>
	    <thumb preview="http://image.tmdb.org/t/p/original/id2qZADRqKEGFxf7yeHLp4tHaEo.jpg">http://image.tmdb.org/t/p/original/id2qZADRqKEGFxf7yeHLp4tHaEo.jpg</thumb>
             <thumb preview="http://image.tmdb.org/t/p/w780/o3P90r3owOlD86e27aC801rk4QI.jpg">http://image.tmdb.org/t/p/original/o3P90r3owOlD86e27aC801rk4QI.jpg</thumb>
</fanart>
```

**Comment by mlaggner** [comment]
which Kodi version do you use?
afaik Kodi 17 slightly changed their NFO style... Maybe this is one change too

**Comment by nathan951** [comment]
This was on 16.1

As soon as I alter the .nfo to the second example and refresh kodi local data the fanart displays properly.

also for poster art this is how kodi save xml for posters:
(You current method works, but there is a difference)
`<thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/inVq3FRqcYIRl2la8iZikYYxFNR.jpg">http://image.tmdb.org/t/p/original/inVq3FRqcYIRl2la8iZikYYxFNR.jpg</thumb>`

I think the preview= has to do with previews for selecting different art. 

On this note i think you should add the option to save additional thumbs to .nfo without locally downloading them.

ie Kodi created .nfo:
 `    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/70yzeQUerxKRxeVE61ob9DlDFjv.jpg">http://image.tmdb.org/t/p/original/70yzeQUerxKRxeVE61ob9DlDFjv.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/rrPWYcAJjx7svXgaoEfTAAxx2Id.jpg">http://image.tmdb.org/t/p/original/rrPWYcAJjx7svXgaoEfTAAxx2Id.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/mXSbp5V3j3Ad7vGrGfFN7xv6cCz.jpg">http://image.tmdb.org/t/p/original/mXSbp5V3j3Ad7vGrGfFN7xv6cCz.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/KY66hTL8YLesWguF1eFhnws9y8.jpg">http://image.tmdb.org/t/p/original/KY66hTL8YLesWguF1eFhnws9y8.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/r8iKqZuYfOGecljTMBGbD4YE9X1.jpg">http://image.tmdb.org/t/p/original/r8iKqZuYfOGecljTMBGbD4YE9X1.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/39z52cy41YViM6tbIohfWYZVi7a.jpg">http://image.tmdb.org/t/p/original/39z52cy41YViM6tbIohfWYZVi7a.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/u6SdQz8Bd45mcuCZdyAFrRCL2AN.jpg">http://image.tmdb.org/t/p/original/u6SdQz8Bd45mcuCZdyAFrRCL2AN.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/1Tco0tfGzXLXTbGZlsyG2L1ME6j.jpg">http://image.tmdb.org/t/p/original/1Tco0tfGzXLXTbGZlsyG2L1ME6j.jpg</thumb>
    <thumb aspect="poster" preview="http://image.tmdb.org/t/p/w500/jMQYcE5ayaQOQUtDQyaDhTa1RL3.jpg">http://image.tmdb.org/t/p/original/jMQYcE5ayaQOQUtDQyaDhTa1RL3.jpg</thumb>
    <fanart>
        <thumb preview="http://image.tmdb.org/t/p/w780/nbIrDhOtUpdD9HKDBRy02a8VhpV.jpg">http://image.tmdb.org/t/p/original/nbIrDhOtUpdD9HKDBRy02a8VhpV.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/n1y094tVDFATSzkTnFxoGZ1qNsG.jpg">http://image.tmdb.org/t/p/original/n1y094tVDFATSzkTnFxoGZ1qNsG.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/4HgqwYTpmrDtafRvjRJ2aOwH69e.jpg">http://image.tmdb.org/t/p/original/4HgqwYTpmrDtafRvjRJ2aOwH69e.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/3P7mVFxpMLblLbVewybYOaNqEHT.jpg">http://image.tmdb.org/t/p/original/3P7mVFxpMLblLbVewybYOaNqEHT.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/mnUbzbYxaaHpaTqFhjl7HwSod7o.jpg">http://image.tmdb.org/t/p/original/mnUbzbYxaaHpaTqFhjl7HwSod7o.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/dfjj7QZBo9JAuMYSbCdIzUWQsr3.jpg">http://image.tmdb.org/t/p/original/dfjj7QZBo9JAuMYSbCdIzUWQsr3.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/iWFx9LVcSQ54TBubh5Fw9G2KrsA.jpg">http://image.tmdb.org/t/p/original/iWFx9LVcSQ54TBubh5Fw9G2KrsA.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/3q7PX4xGF4YkDzOpcQXLCIYXHFK.jpg">http://image.tmdb.org/t/p/original/3q7PX4xGF4YkDzOpcQXLCIYXHFK.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/l5003rpuMgaqzkw6QpZ1F3k9pKA.jpg">http://image.tmdb.org/t/p/original/l5003rpuMgaqzkw6QpZ1F3k9pKA.jpg</thumb>
        <thumb preview="http://image.tmdb.org/t/p/w780/iPlhdbPq2CwoOtD2bKDAACoSmGV.jpg">http://image.tmdb.org/t/p/original/iPlhdbPq2CwoOtD2bKDAACoSmGV.jpg</thumb>
    </fanart>`


Lastly on the subject of downloaded art, is there anyway to set kodi to use the TMM downloaded art in .nfo file or is the downloaded fan-art unusable? 

ie
`<thumb preview="\\fanart.jpg">\\fanart.jpg</thumb>`

Thanks,
Nathan

**Comment by mlaggner** [comment]
kodi ingores artwork tags in the NFO if there is local artwork found (thus we did not saw that difference for a while).
If you download artwork by tmm, you don't need any <thumb> tags in the NFO to let kodi find it

I'll have a look how to change the nfo structure without destroying too much, but since that is mostly unneeded, this may take a while ;)

**Comment by nathan951** [comment]
> kodi ingores artwork tags in the NFO if there is local artwork found (thus we did not saw that difference for a while).
> If you download artwork by tmm, you don't need any tags in the NFO to let kodi find it

Unfortunately if I have TMM create .nfo files, kodi will use the .nfo for importing data/thumbs even if i preselect source to grab it from TMDB. This is a problem because the fanart XML tag is incorrect so the movie is imported without any fanart, because of TMM. Now if i tell TMM NOT to create nfo this is not a problem, as it grabs the poster and fanart from TMDB not the incorrect format from TMM, then kodi can create the correct XML on export (multi-file).

In short kodi sees TMM .nfo with a empty fanart tag (ie <fanart></fanart>) because it's improperly formatted and completely missing the <thumb> within) 

incorrect:
`<fanart>http://image.tmdb.org/t/p/original/id2qZADRqKEGFxf7yeHLp4tHaEo.jpg</fanart>`

correct:
`<fanart>
     <thumb preview="http://image.tmdb.org/t/p/original/id2qZADRqKEGFxf7yeHLp4tHaEo.jpg">http://image.tmdb.org/t/p/original/id2qZADRqKEGFxf7yeHLp4tHaEo.jpg</thumb>
      <thumb preview="http://image.tmdb.org/t/p/w780/o3P90r3owOlD86e27aC801rk4QI.jpg">http://image.tmdb.org/t/p/original/o3P90r3owOlD86e27aC801rk4QI.jpg</thumb>
</fanart>`

**Comment by nathan951** [comment]
has this been addressed?

Thanks,
Nathan

**Comment by mlaggner** [comment]
yes, but only for v3 since this is a breaking NFO structure change
